### PR TITLE
Add metrics test to statuspage

### PR DESCRIPTION
**Pull request by MaksTkachuk**
+fix a bug with naming in sanbase_webapp.py
+formatting

### CODE REVIEW

File: api_tests/statuspage/metrics.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        message = test_metrics()
+    except (RequestError, RequestException) as e:
+        message = str(e)
+        logging.error(message)
+    else:
+        logging.info('Success')
+
+    if config.getboolean('send_discord_notification') and message:
+        logging.info('Sending discord notification...')
+        publish_sanbase_webapp_alert(message)
```
Comment (kalkov) [inline api_tests/statuspage/metrics.py:36]: Shouldn't be `publish_metrics_alert`?

File: api_tests/statuspage/metrics.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    result_json = latest_json()
+    slug_report = {}
+    severity_ratio = 0.0
+    for slug in STATUSPAGE_SLUGS:
+        slug_report['slug'] = get_errors_metrics_names(result_json[slug])
+        severity_ratio += calculate_severity_ratio(
+            result_json[slug])/len(STATUSPAGE_SLUGS)
+    broken_metrics = set([*slug_report[slug] for slug in slug_report])
+
+    incidents = list(filter(filter_open_sanbase_webapp_incident,
```
Comment (kalkov) [inline api_tests/statuspage/metrics.py:54]: This function will be used both by webapp and metrics?

File: api_tests/statuspage/metrics.py (hunk 1 above)
Comment (MaksTkachuk) [reply api_tests/statuspage/metrics.py]: leftover

File: api_tests/statuspage/metrics.py (hunk 2 above)
Comment (MaksTkachuk) [reply api_tests/statuspage/metrics.py]: leftover

File: app.py [hunk 3]
```diff
@@ -5,10 +5,10 @@
 from flask_bootstrap import Bootstrap
 import boot
 from datetime import datetime as dt
+from api_tests.models.queries import get_test_suite
```
Comment (kalkov) [inline app.py:8]: Use a single import:

```python
from api_tests.models.queries import get_test_suite, get_latest_test_suite ...
```

**Review by kalkov — APPROVED** [review]

**Pull request merged**